core/templates/css/oppia.css

```diff
@@ -4516,3 +4524,211 @@ div.oppia-issues-learner-action {
 .oppia-rule-preview {
   word-wrap: break-word;
 }
+
+.oppia-welcome-guide-block {
+  font-family: "Roboto", Arial, sans-serif;
+  font-size: 1.17em;
+  line-height: 128%;
+  margin-bottom: 10px;
+  text-align: left;
+  width: 100%;
+}
+
+.oppia-welcome-guide-second-section {
+  padding: 40px 0 40px 70px;
+}
+
+.oppia-welcome-guide-third-section {
+  padding: 40px 70px 0px 70px;
+}
+
+.oppia-welcome-guide-fractions-image {
+  margin: auto;
+  padding-bottom: 40px;
+  text-align: center;
+  width: auto;
+}
+
+@media (max-width: 768px) {
+  .oppia-welcome-guide-body {
+    padding: 20px 0 0 20px;
+  }
+}
+
+@media (min-width: 992px) {
+  .oppia-welcome-guide-img {
+    height: 460px;
+    text-align: right;
+  }
+
+  .oppia-welcome-guide-header {
+    height: 460px;
+    padding: 80px 20px 0 70px;
+  }
+
+  .oppia-welcome-guide-img img {
+    width: auto;
+    height: 460px;
```

Comment:
Add this in sorted order of property name*

Reply:
Do these changes in other places (if required)

Reply:
fixed